src/providers/on-chain-quote-provider.ts

```diff
@@ -424,12 +430,15 @@ export class OnChainQuoteProvider implements IOnChainQuoteProvider {
     );
 
     metric.putMetric(
-      `${this.metricsPrefix}QuoteBatchSize`,
+      `${this.metricsPrefix(this.chainId, useMixedRouteQuoter)}QuoteBatchSize`,
       inputs.length,
       MetricLoggerUnit.Count
     );
     metric.putMetric(
-      `${this.metricsPrefix}QuoteBatchSize_${ID_TO_NETWORK_NAME(this.chainId)}`,
+      `${this.metricsPrefix(
+        this.chainId,
+        useMixedRouteQuoter
+      )}QuoteBatchSize_${ID_TO_NETWORK_NAME(this.chainId)}`,
```

Comment:
should we store the metric prefix in a `const` and then use that `const`?

Reply:
It might be slightly difficult, because the metrix prefix now contains the chain id, which is dynamic